// invalid number of results (of built-in function) when used in call expression (expecting 1, getting 0)

print(stop("error"));